tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38-{flake8,black,test,pylint}
+envlist = py38, flake8, black, test, pylint
```

Comment:
What is the motivation for this change?

Reply:
Sorry, It is a remnant of testing with Python 3.10.
I will revert it.

Reply:
I reverted it. (https://github.com/mfranberg/libplinkio/pull/34/commits/72b5ee44d0ec713668a38483252378c6408d8ce5)